servers/rendering/renderer_rd/renderer_scene_render_rd.h

```diff
@@ -53,12 +53,18 @@ struct RenderDataRD {
 
 	Transform3D cam_transform;
 	CameraMatrix cam_projection;
+	Vector2 taa_jitter;
 	bool cam_orthogonal = false;
 
 	// For stereo rendering
 	uint32_t view_count = 1;
 	CameraMatrix view_projection[RendererSceneRender::MAX_RENDER_VIEWS];
 
+	Transform3D prev_cam_transform;
```

Comment:
Since this is now stored in RenderData, we no longer need to keep a copy in ``rb->volumetric_fog`` we should remove it to keep things clean.

See 
https://github.com/godotengine/godot/blob/8a21f7231f0851aec4c06abee28a549b1f8083fd/servers/rendering/renderer_rd/renderer_scene_render_rd.h#L797
and
https://github.com/godotengine/godot/blob/8a21f7231f0851aec4c06abee28a549b1f8083fd/servers/rendering/renderer_rd/renderer_scene_render_rd.cpp#L4750-L4751
and
https://github.com/godotengine/godot/blob/8a21f7231f0851aec4c06abee28a549b1f8083fd/servers/rendering/renderer_rd/renderer_scene_render_rd.cpp#L4272-L4273
and
https://github.com/godotengine/godot/blob/8a21f7231f0851aec4c06abee28a549b1f8083fd/servers/rendering/renderer_rd/renderer_scene_render_rd.cpp#L4837